doc/development/deprecations.rst

```diff
@@ -9,6 +9,41 @@ deprecations are listed below.
 Pending deprecations
 --------------------
 
+New operator arithmetic deprecations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This release completes the main phase of PennyLane's switchover to an updated approach for handling
+arithmetic operations between operators, check out the :ref:`Updated operators <new_opmath>` page
+for more details. The old system is still accessible via :func:`~.disable_new_opmath`. However, the
+old system will be removed in an upcoming release and should be treated as deprecated. The following
+functionality will explicitly raise a deprecation warning when used:
+
+* ``op.ops`` and ``op.coeffs`` will be deprecated in the future. Use ``op.terms()`` instead.
+
+  - Added and deprecated for ``Sum`` and ``Prod`` instances in v0.35
+
+* Accessing ``qml.ops.Hamiltonian`` with new operator arithmetic enabled is deprecated. Using ``qml.Hamiltonian``
+  with new operator arithmetic enabled now returns a ``LinearCombination`` instance. Some functionality
+  may not work as expected, and use of the Hamiltonian class with the new operator arithmetic will not
+  be supported in future releases of PennyLane.
+
+  You can update your code to the new operator arithmetic by using ``qml.Hamiltonian`` instead of importing
+  the Hamiltonian class directly or via ``qml.ops.Hamiltonian``. When the new operator arithmetic is enabled,
+  ``qml.Hamiltonian`` will access the new corresponding implementation.
+
+  Alternatively, to continue accessing the legacy functionality, you can use
+  ``qml.operation.disable_new_opmath()``.
+
+  - Deprecated in v0.36
```

Comment:
I'm a little confused by this 🤔. Everything seemed fine up to

> Alternatively, to continue accessing the legacy functionality, you can use ``qml.operation.disable_new_opmath()``.

> Deprecated in v0.36.

How long can I continue to use the legacy functionality?

Reply:
```suggestion
* Accessing ``qml.ops.Hamiltonian`` with new operator arithmetic enabled is deprecated. Using ``qml.Hamiltonian``
  with new operator arithmetic enabled now returns a ``LinearCombination`` instance. Some functionality
  may not work as expected, and use of the Hamiltonian class with the new operator arithmetic will not
  be supported in future releases of PennyLane.

  You can update your code to the new operator arithmetic by using ``qml.Hamiltonian`` instead of importing
  the Hamiltonian class directly or via ``qml.ops.Hamiltonian``. When the new operator arithmetic is enabled,
  ``qml.Hamiltonian`` will access the new corresponding implementation.

  Alternatively, to continue accessing the legacy functionality, you can use
  ``qml.operation.disable_new_opmath()``.

  - Use of ``qml.ops.Hamiltonian`` is deprecated in v0.36
```
Does this clarify?